packages/elements/src/dialog/index.ts

```diff
@@ -101,11 +114,38 @@ export class Dialog extends Overlay {
     `];
   }
 
+  private _header: string | null = null;
   /**
-   * Set Header/Title of the dialog
+   * Header element
+   * @param value Header text
    */
   @property({ type: String })
-  public header: string | null = null;
+  public set header (value: string | null) {
+    const oldValue = this._header;
+    const ariaLabelledby = this.getAttribute('aria-labelledby');
+    if(oldValue !== value) {
+      if(this.ariaLabel.length === 0 && ariaLabelledby == null) {
```

Comment:
Use this condition could be enough.
```suggestion
      if(!this.ariaLabel && !ariaLabelledby) {
```

Reply:
I have already fixed it